### PR TITLE
Sdk roll failure - add facet to allowlist test_manager and other packages.

### PR DESCRIPTION
**Pull request by naudzghebre**

### CODE REVIEW

File: shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart_jit_echo_server/BUILD.gn [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
```
Comment (lin-erik) [inline shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart_jit_echo_server/BUILD.gn:1]: I don't think this file should exist, maybe merge conflict?

**Review by lin-erik — CHANGES_REQUESTED** [review]

File: shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart_jit_echo_server/BUILD.gn (hunk 1 above)
Comment (naudzghebre) [reply shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart_jit_echo_server/BUILD.gn:1]: It wouldn't let me push the change without formatting it. Not sure how to get around it.

Comment (lin-erik) [reply shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart_jit_echo_server/BUILD.gn:1]: If you delete it, the linter still complains? What.. 😭

Comment (naudzghebre) [reply shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart_jit_echo_server/BUILD.gn:1]: Oh wait, you mean the entire file just doesn't belong in flutter/engine ?

**Review by lin-erik — APPROVED** [review]

**Comment by naudzghebre** [comment]
fixes#114820

**Comment by gaaclarke** [comment]
FYI this has broken the tree, here is the failure: https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20Fuchsia%20FEMU/12755/overview

This will probably be reverted by the sheriff.

**Pull request merged**